zipline-gradle-plugin/src/main/kotlin/app/cash/zipline/gradle/ZiplinePlugin.kt

```diff
@@ -73,10 +70,11 @@ class ZiplinePlugin : KotlinCompilerPluginSupportPlugin {
       createdTask.outputDir.set(linkOutputDir.parentFile.resolve("${linkOutputDir.name}Zipline"))
     }
 
-    project.tasks.withType(KotlinWebpack::class.java).configureEach { kotlinWebpack ->
-      if (kotlinBinary.mode.toString().equals(kotlinWebpack.mode.toString(), ignoreCase = true)) {
-        kotlinWebpack.dependsOn(compileZiplineTaskName)
-      }
+    val serveTaskName = "serve${kotlinBinary.mode.toString().lowercase().capitalized()}Zipline"
```

Comment:
Both `lowercase()` and `capitalized()` should be given `Locale.US` so that we do not create different task names based on the user's locale.

Also I think you want `.name` not `.toString()`, assuming the `mode` is an enum. `toString()` defaults to `name`, but `toString()` also can be customized whereas `name` cannot be (other than renaming the constant, of course).

Reply:
Thanks! Apparently, I was using the Gradle version of capitalized which doesn't take a Locale, and the Kotlin one is deprecated, so I'll just use replaceFirstChar here.

Reply:
Yeah the Kotlin replacement is a bit pedantic. I hate how verbose it is.